apps/openchallenges/organization-service/src/test/java/org/sagebionetworks/openchallenges/organization/service/exception/GlobalExceptionHandlerTest.java

```diff
@@ -0,0 +1,54 @@
+package org.sagebionetworks.openchallenges.organization.service.exception;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Locale;
+import org.junit.jupiter.api.Test;
+import org.sagebionetworks.openchallenges.organization.service.model.dto.BasicErrorDto;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+public class GlobalExceptionHandlerTest {
+
+  @Test
+  public void GlobalExceptionHandler_ShouldReturnStatus_WhenExceptionAndLocalePassed() {
+    // Create a sample Exception
+    Exception exception = new Exception("An exception occurred");
+    Locale locale = Locale.getDefault();
+    String details = "Something went wrong";
+
+    // Create the GlobalExceptionHandler instance
+    GlobalExceptionHandler exceptionHandler = new GlobalExceptionHandler();
+
+    // Call handleException
+    ResponseEntity<BasicErrorDto> responseEntity =
+        exceptionHandler.handleException(exception, locale);
+
+    // Verify the response
+    assertThat(responseEntity.getStatusCode()).isEqualTo(HttpStatus.INTERNAL_SERVER_ERROR);
+  }
+
+  @Test
+  public void HandleGlobalException_ShouldFunction_WhenCalled() {
+    // Create a sample Exception
+    // Exception exception = new Exception("An exception occurred");
+    Locale locale = Locale.getDefault();
+    // String details = "Something went wrong";
+    // Define the exception details
+    String type = "ExceptionType";
+    String title = "Exception Title";
+    HttpStatus status = HttpStatus.BAD_REQUEST;
+    String detail = "Exception detail message";
+
+    // Create newSimpleChallengeGlobalException instance
+    SimpleChallengeGlobalException exception =
+        new SimpleChallengeGlobalException(type, title, status, detail);
+
+    // Create the GlobalExceptionHandler instance
+    GlobalExceptionHandler exceptionHandler = new GlobalExceptionHandler();
+
+    // Call handleGlobalException
+    ResponseEntity<BasicErrorDto> responseEntity =
+        exceptionHandler.handleGlobalException(exception, locale);
```

Comment:
As discussed, you need to assert `responseEntity`

Reply:
Assertion of responseEntity included